src/App.css

```diff
@@ -33,5 +36,103 @@ body {
   padding: 2px;
 }
 h1 {
-  font-size: 20px;
+  font-size: 20px; 
+  font-weight: 600;
+  border-bottom: 2px solid #090707;
+}
+.friends {
+  width: auto;  
+  height: 100px;
+  color: #070300;
+}
+
+.list-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0; 
+  border-top: 2px solid #27221e;
+  padding: 5px;
+  margin: 0px -10px 0px -10px;
+}
+
+.friend-name-container {
+  padding: 5px;
+  margin-left: 10px;
+}
+
+
+.search-friend {
+  background: transparent;
+  border: 0;
+  box-shadow: none;
+}
+.search-label {
+  border-bottom: 2px solid #504444;
+  margin-left: 10px;
+  min-width: 120px;
+  color: #594c4e;
+}
+
+.add-friend-input {
+  width: 103%;
+  box-sizing: border-box;
+  padding: 17px;
+  font-size: 14px;
+  border: 0;
+  color: #4e4142;
+  margin-left: -10px;
+}
+
+.delete-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0; 
+  border-top: 2px solid #27221e;
+  padding: 18px 5px 0px 0px;
+  margin: 0px -10px 0px -10px;
+  vertical-align: baseline;
+}
+
+.delete-container-p {
+  padding: 5px;
+  margin-left: 10px;
+}
+
+button {
+  margin: 0px 15px 0px -5px;
+  border: 2px solid #8d7b6b;
+  border-radius: 3px;
+  padding: 5px 10px;
+  background: #b3a59a;
+}
+
+p {
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+  font-size: 14px;
+  color: #0d090a;
+}
+
+h2 {
+  font-size: 18px;
+  font-weight: 600;
+  margin-bottom: -10px;
```

Comment:
negative margin

Reply:
without that property heading(name) and paragraph (is your friend) were far apart.